bin/git-get

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
```

Comment:
Let's add this cmd in https://github.com/tj/git-extras/blob/master/not_need_git_repo